src/testUtils/index.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import {promises as fs} from 'fs';
+
+export function removeFileExtension(fileName) {
+  return fileName.substr(0, fileName.lastIndexOf('.'));
+}
+
+function removeUnknown(e) {
+  return e !== 'Unknown';
+}
+
+export async function getStatesAndDistrictsFromAPI() {
```

Comment:
A snippet of the output. We get an object with keys set to the name of the state and the districts of the state as values.
```javascript
{
        'Arunachal Pradesh': [ 'Lohit' ],
        'Dadra and Nagar Haveli': [],
        Tripura: [ 'Gomati', 'North Tripura' ]
}
```